write-fonts/src/tables/glyf.rs

```diff
@@ -0,0 +1,298 @@
+//! The [glyf (Glyph Data)](https://docs.microsoft.com/en-us/typography/opentype/spec/glyf) table
+
+use kurbo::{BezPath, Shape};
+
+use read_fonts::tables::glyf::{CurvePoint, SimpleGlyphFlags};
+
+use crate::FontWrite;
+
+/// A single quadratic bezier contour
+#[derive(Clone, Debug)]
+pub struct Contour(Vec<CurvePoint>);
+
+/// A simple (without components) glyph
+pub struct SimpleGlyf {
+    x_max: i16,
+    y_max: i16,
+    x_min: i16,
+    y_min: i16,
+    contours: Vec<Contour>,
+    _instructions: Vec<u8>,
+}
+
+/// An error if an input curve is malformed
+#[derive(Clone, Debug)]
+pub enum BadKurbo {
+    HasCubic,
+    TooSmall,
+    MissingMove,
+}
+
+/// A helper trait for converting other point types to open-type compatible reprs
+pub trait OtPoint {
+    fn get(self) -> (i16, i16);
+}
+
+impl OtPoint for kurbo::Point {
+    fn get(self) -> (i16, i16) {
+        (ot_round(self.x as f32), ot_round(self.y as f32))
+    }
+}
+
+impl OtPoint for (i16, i16) {
+    fn get(self) -> (i16, i16) {
+        self
+    }
+}
+
+// adapted from simon:
+// https://github.com/simoncozens/rust-font-tools/blob/105436d3a617ddbebd25f790b041ff506bd90d44/otmath/src/lib.rs#L17
+fn ot_round(val: f32) -> i16 {
+    (val + 0.5).floor() as i16
+}
```

Comment:
Is glyf the only place that performs ot-round's? If not perhaps shift to a helper library/common location?

Reply:
yea maybe this moves to font-types eventually?

Reply:
I suggest we follow the lead of https://github.com/googlefonts/fontations/pull/221 and move it to font-types nowish?